contrib/ros/README.md

```diff
@@ -0,0 +1,138 @@
+#### INSTALL INTERA ROBOT SDK
+##### Activate the conda environment for rllab
+    $ source activate rllab3
+
+**rllab3 is your rllab conda environment's name** 
+##### Download the SDK on your Workstation
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src
+    $ wstool init .
+    $ git clone https://github.com/RethinkRobotics/sawyer_robot.git
+    $ wstool merge sawyer_robot/sawyer_robot.rosinstall
+    $ wstool update
+##### Download the dependecies 
+    $ git clone https://github.com/ros/geometry.git
+    $ cd geometry
+    $ git checkout indigo-devel
+##### Source ROS Setup
+    $ source /opt/ros/kinetic/setup.bash
+##### Build
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ catkin_make -DPYTHON_EXECUTABLE:FILEPATH=<path_to_your_rllab_env_python_interp>
+#### GONFIGURE ROBOT COMMUNICATION/ROS WORKSPACE
+##### intera.sh ROS Environment Setup
+##### Copy the intera.sh script
+The intera.sh file already exists in intera_sdk repo, copy the file into your ros workspace.
+    
+    $ cp ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src/intera_sdk/intera.sh ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+##### Customize the intera.sh script
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ gedit intera.sh
+##### Edit the 'robot_hostname' field
+##### Edit the 'your_ip' field
+##### Verify 'ros_version' field
+update: ros_version='kinetic'
+##### Save and Close intera.sh script
+##### Initialize your SDK environment
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ ./intera.sh
+### Setup Gazebo Simulation for Sawyer
+**Please skip this section if you don't want simulation**
+#### Installation/Prerequisites
+- Make sure you have finished [workstation setup](#setup-workstation)
+- Ensure the following software packages are installed:
+    
+    
+    $ sudo apt-get install gazebo7 ros-kinetic-qt-build ros-kinetic-gazebo-ros-control ros-kinetic-gazebo-ros-pkgs ros-kinetic-ros-control ros-kinetic-control-toolbox ros-kinetic-realtime-tools ros-kinetic-ros-controllers ros-kinetic-xacro python-wstool ros-kinetic-tf-conversions ros-kinetic-kdl-parser ros-kinetic-sns-ik-lib
+#### Sawyer Simulator Installation
+##### Install sawyer_simulator
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src
+    $ git clone https://github.com/RethinkRobotics/sawyer_simulator.git
+    $ wstool merge sawyer_simulator/sawyer_simulator.rosinstall
+    $ wstool update
+##### Build Source
+    $ source /opt/ros/kinetic/setup.bash
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ catkin_make -DPYTHON_EXECUTABLE:FILEPATH=<path_to_your_rllab_env_python_interp>
+### Convert Python2 code in intera packages to Python3
+- As our environments only support Python3, we need to convert Python2 code in intera packages.
+- These can be done using rllab.contrib.ros.scripts.sawyer_2to3.sh
+##### Customize sawyer_2to3.sh
+Upgrade path:
+    
+    ros_ws=<YOUR_FAVORITE_WORKSPACE_PATH>/src/
+Remove sawyer_simulator, if you don't need it:
+
+    packages=(intera_sdk)
+##### Execute script
```

Comment:
Please be very specific about where to ``cd`` into before executing the script, and provide a code example.

Reply:
./sawyer_2to3.sh

If permission denied, run this: 
$ sudo chmod +x sawyer_2to3.sh

Reply:
What I added:
```bash
$ rllab_path=<the_path_to_rllab> ./$rllab_path/contrib/ros/scripts/sawyer_2to3.sh
```